plugins/inputs/prometheus/start.go

```diff
@@ -79,6 +83,43 @@ func init() {
 	prometheus.MustRegister(v.NewCollector("prometheus"))
 }
 
+// @TODO: REMOVE BEFORE RELEASE
+func debugScrapeManager(logger log.Logger, scrapeManager *scrape.Manager) {
+	for {
+		for key, targets := range scrapeManager.TargetsAll() {
+			level.Info(logger).Log("msg", "ScraperDebug", "key", key)
+			for _, target := range targets {
+				level.Info(logger).Log("msg", "ScraperDebug-Tar", "target", target.String(), "lastScrape", target.LastScrape())
+			}
+		}
+		time.Sleep(5 * time.Second)
+	}
+}
+
+// @TODO: REMOVE BEFORE RELEASE
+func debugChannelWrapper(logger log.Logger, in <-chan map[string][]*targetgroup.Group, out chan<- map[string][]*targetgroup.Group) {
```

Comment:
Do we need these functions ?

Reply:
We use these function for debugging during development so I wanted to keep them since we are merging to a dev branch and not main